Add a note about Mock classes not considered for security issues (#18730)
symfony/symfony-docs: contributing/code/security.rst

  production (including the web profiler or anything enabled when ``APP_DEBUG``
  is set to ``true`` or ``APP_ENV`` set to anything but ``prod``);

> * Any security issues found in classes provided to help fo testing that should

Reviewer: ```suggestion
* Any security issues found in classes provided for testing that should
```

Reviewer 2: Fixed while merging. Thanks.

  never be used in production (like for instance mock classes that contain

>   ``Mock`` in their name);

Reviewer 3: I suggest mentioning the `Test` namespace as well inside those braces.

Reviewer 2: Added while merging. See https://github.com/symfony/symfony-docs/commit/ea432f9c2516b7c89662eaf6bf83ff6efaeafc81 Thanks.
